orchestration/roles/app/tasks/setup_virtualenv.yml

```diff
@@ -3,17 +3,37 @@
 - name: Ensure virtualenvwrapper is installed
   pip: name=virtualenvwrapper executable="/usr/bin/pip3"
 
-- name: Export WORKON_HOME in .bashrc
-  lineinfile: dest=/home/{{ gunicorn_user }}/.bashrc line="export WORKON_HOME=$HOME/.virtualenvs"
+- name: Export WORKON_HOME in /root/.bashrc
+  lineinfile: dest=/root/.bashrc line="export WORKON_HOME=/server/.virtualenvs"
+  when: env != "vagrant"
 
-- name: Export PROJECT_HOME in .bashrc
-  lineinfile: dest=/home/{{ gunicorn_user }}/.bashrc line="export PROJECT_HOME=$HOME/Devel"
+- name: Export PROJECT_HOME in /root/.bashrc
+  lineinfile: dest=/root/.bashrc line="export PROJECT_HOME=/server/Devel"
+  when: env != "vagrant"
 
-- name: Export VIRTUALENVWRAPPER_PYTHON in .bashrc
-  lineinfile: dest=/home/{{ gunicorn_user }}/.bashrc line="export VIRTUALENVWRAPPER_PYTHON=/usr/bin/python3"
+- name: Export VIRTUALENVWRAPPER_PYTHON in /root/.bashrc
+  lineinfile: dest=/root/.bashrc line="export VIRTUALENVWRAPPER_PYTHON=/usr/bin/python3"
+  when: env != "vagrant"
 
 - name: Source virtualenvwrapper.sh in .bashrc
-  lineinfile: dest=/home/{{ gunicorn_user }}/.bashrc line="source /usr/local/bin/virtualenvwrapper.sh"
+  lineinfile: dest=/root/.bashrc line="source /usr/local/bin/virtualenvwrapper.sh"
+  when: env != "vagrant"
+
+- name: Export WORKON_HOME in /root/.bashrc
+  lineinfile: dest=/home/vagrant/.bashrc line="export WORKON_HOME=/server/.virtualenvs"
```

Comment:
instead of having a different ansible command for each of these, could we just set the destination of .bashrc as a variable that's different in vagrant than on our deployed servers?